test/integration/api.bats

```diff
@@ -172,9 +198,24 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker rename" {
```

Comment:
Actually, I think we don't really support renaming in Swarm (this is a new feature).

We should definitely upstate the `Store` with the new name.